cost-analyzer/values.yaml

```diff
@@ -932,6 +932,43 @@ kubecostDeployment:
     configVolumeSize: 1Gi
     initImage: {}
 
+kubecostAggregator:
+  cloudCost:
+    enabled: false
+  replicas: 0
```

Comment:
Why not an "enabled" switch instead of basing this off of replicas with a default of `0`?

Reply:
no good reason :) fixed, that cleans things up nicely, good looks